Remove defunct rgb_operm

ffontaine opened the pull request:
[Retrieved (and slightly updated) from:
https://github.com/eddelbuettel/dieharder/pull/2/commits/40d377b86c856f5a4510a6f5cd56be004873ad77]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

eddelbuettel commented:
As this is (at least in spirit) still Robert's baby I would prefer not to _delete_ but simply disable (if and when we see the need to do so).  Would you mind sticking the sources back in, but keeping them commented out in the Makefile and test runner as you have?

ffontaine committed:
Remove defunct rgb_operm

[Retrieved (and slightly updated) from:
https://github.com/eddelbuettel/dieharder/pull/2/commits/40d377b86c856f5a4510a6f5cd56be004873ad77]

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

ffontaine commented:
I updated the PR as requested

eddelbuettel reviewed (approved):
Thank you!
